README.md

```diff
@@ -1,23 +1,14 @@
-Adafruit Si7021
+Si7021 Temperature/Humidity Sensor
```

Comment:
dont edit the readme

Reply:
Yes all of these readme/doc link should not be merged into your repo,  The reason I put those changes in was that I wanted all problems with my repo directed to me instead of having people blame you for my errors.  It you could just look at the .cpp and .h everything else should not be accepted.